dist/common/scripts/scylla_raid_setup

```diff
@@ -157,11 +231,27 @@ if __name__ == '__main__':
 
     os.makedirs(mount_at, exist_ok=True)
 
-    uuid = out(f'blkid -s UUID -o value {fsdev}')
-    if not uuid:
-        raise Exception(f'Failed to get UUID of {fsdev}')
-
-    uuidpath = f'/dev/disk/by-uuid/{uuid}'
+    udev_info = UdevInfo(fsdev)
+    mount_dev = None
+    if udev_info.uuid_link:
+        mount_dev = udev_info.uuid_link
+    else:
+        if udev_info.label_link:
+            mount_dev = udev_info.label_link
+            dev_type = 'label'
+        if udev_info.partuuid_link:
```

Comment:
should this be 'if ... elif... elif...' ?

Reply:
Changed to elif.